src/config.rs

```diff
@@ -248,35 +248,29 @@ fn read_devices(
     Ok(devices)
 }
 
-fn locate_fragments(root: &Path) -> BTreeMap<String, PathBuf> {
-    let base_dirs = vec![
-        String::from(root.join("usr/lib").to_str().unwrap()),
-        String::from(root.join("usr/local/lib").to_str().unwrap()),
-        String::from(root.join("etc").to_str().unwrap()),
-        String::from(root.join("run").to_str().unwrap()), // We look at /run to allow temporary overriding
-                                                          // of configuration. There is no expectation of
-                                                          // programatic creation of config there.
+fn locate_fragments(root: &Path) -> BTreeMap<OsString, PathBuf> {
+    let base_dirs = [
+        root.join("usr/lib"),
+        root.join("usr/local/lib"),
+        root.join("etc"),
+        root.join("run"), // We look at /run to allow temporary overriding
```

Comment:
This could use https://github.com/coreos/liboverdrop-rs/pull/26 in the future too.